eng/pipelines/templates/Apex_Tests_On_Windows.yml

```diff
@@ -41,11 +41,12 @@ stages:
           - ${{parameters.variables}}
       runSettingsURI: ${{parameters.runSettingsURI}}
       visualStudioBootstrapperURI: $(bootstrapperUrl)      
-      baseVisualStudioBootstrapperURI: ${{parameters.baseBuildDrop}};bootstrappers/Enterprise/vs_enterprise.exe
+      baseVisualStudioBootstrapperURI: https://vsdrop.corp.microsoft.com/file/v1/$(VisualStudio.BaseBuild.ProductsDropName);bootstrappers/Enterprise/vs_enterprise.exe
       candidateBaselineBuilds: $(BaselineBuildCommitIds)
       testLabPoolName: VS-Platform
       dartLabEnvironment: ${{parameters.DartLabEnvironment}}
       visualStudioSigning: Test
+      testMachineConfigurationJobTimeoutInMinutes: 30
```

Comment:
Related to timeout https://github.com/NuGet/NuGet.Client/pull/5283/files#r1253811132